### PR TITLE
Lab 6

### PR DESCRIPTION
**Pull request by corey-buchanan**

### CODE REVIEW

File: .DS_Store
Comment (warnerjon12) [inline .DS_Store:1]: This file should not have been committed. I usually run `git status` prior to committing just to make sure I am committing what I think I am, especially if using `git commit -a`

File: src/hello_freertos.c
Comment (warnerjon12) [inline src/hello_freertos.c:1]: This looks good, but I would add tests to verify that this does what you think it does for binary and mutex semaphores, respectively. You could probably refactor this code into test code by replacing (or supplementing) the print statements with something you can check and making the supervisor task a test function.

**Review by warnerjon12 — CHANGES_REQUESTED** [review]
Overall looks good, but please remove .DS_Store and write tests for activity 0

File: .DS_Store
Comment (Mgomez-01) [reply .DS_Store:1]: removed

**Review by chasegriswold — APPROVED** [review]
Looks good to me, and CICD is passing. I think it's ready to merge.

**Pull request merged**